### PR TITLE
added version flag, for testability

### PR DESCRIPTION
**Pull request by uforic**
I created a command called version which print the version of godefinfo, and but primarily exists to have an exit code of 0 for testability.

### CODE REVIEW

File: godefinfo.go [hunk 1]
```diff
@@ -54,6 +55,10 @@ func main() {
 		flag.Usage()
 		os.Exit(2)
 	}
+	if *version {
+		fmt.Fprintf(os.Stdout, "godefino version 0.1")
```
Comment (sqs) [inline godefinfo.go]: Typo godefinfo

Comment (sqs) [inline godefinfo.go]: fmt.Printf not Fprintf (unnecessary for stdout)

**Comment by uforic** [comment]
Tested against godefinfo_test, Printf used instead of Fprintf, and typo fixed. Thanks for looking over!

File: godefinfo.go [hunk 2]
```diff
@@ -54,6 +55,10 @@ func main() {
 		flag.Usage()
 		os.Exit(2)
 	}
+	if *version {
+		fmt.Printf("godefinfo version 0.1")
```
Comment (sqs) [inline godefinfo.go]: Should have \n or use Println

File: godefinfo.go [hunk 3]
```diff
@@ -29,6 +29,7 @@ var (
 	filename  = flag.String("f", "", "Go source filename")
 	gobuild   = flag.Bool("gobuild", false, "automatically run `go build -i` on the filename to rebuild deps (necessary for cross-package lookups)")
 	importsrc = flag.Bool("importsrc", true, "import external Go packages from source (can be slower than -gobuild)")
+	version   = flag.Bool("v", false, "Version of godefinfo")
```
Comment (sqs) [inline godefinfo.go]: Lowercase v in Version

**Comment by uforic** [comment]
Taken care of!

**Pull request merged**